lib/sys/sys+osx-x64.myr

```diff
@@ -1081,6 +1096,14 @@ const sysctl = {mib, old, oldsz, new, newsz
 		(mib : int#), a(mib.len), old, oldsz, new, newsz) : int)
 }
 
+const ulock_wait = {op, uaddr, val, timeout
```

Comment:
If you can wrap syscall args with the `a()` function, it will ensure it's cast to an appropriate type. It shouldn't matter here, but there can be some issues with unset top bits with smaller argument values.

Reply:
Done.